erpnext/hr/doctype/hr_settings/hr_settings.json

```diff
@@ -22,6 +22,7 @@
   "email_salary_slip_to_employee",
   "encrypt_salary_slips_in_emails",
   "password_policy",
+  "allow_salary_accrual_per_employee",
```

Comment:
The payroll module is merged please rebase your PR and push again.
and move the setting to new Payroll settings.

Reply:
Completed